backend/laundry/urls.py

```diff
@@ -1,13 +1,19 @@
 from django.urls import path
+from django.views.decorators.cache import cache_page
 
 from laundry.views import HallInfo, HallUsage, Ids, MultipleHallInfo, Preferences, Status
+from utils.cache import MINUTE_IN_SECONDS, MONTH_IN_SECONDS_APPROX
 
 
 urlpatterns = [
-    path("hall/<hall_id>/", HallInfo.as_view(), name="hall-info"),
-    path("usage/<hall_id>/", HallUsage.as_view(), name="hall-usage"),
-    path("rooms/<hall_ids>", MultipleHallInfo.as_view(), name="multiple-hall-info"),
-    path("halls/ids/", Ids.as_view(), name="hall-ids"),
-    path("status/", Status.as_view(), name="status"),
+    path("hall/<hall_id>/", cache_page(MINUTE_IN_SECONDS)(HallInfo), name="hall-info",),
+    path("usage/<hall_id>/", cache_page(MINUTE_IN_SECONDS)(HallUsage), name="hall-usage",),
+    path(
+        "rooms/<hall_ids>",
+        cache_page(MINUTE_IN_SECONDS)(MultipleHallInfo),
+        name="multiple-hall-info",
+    ),
+    path("halls/ids/", cache_page(MONTH_IN_SECONDS_APPROX)(Ids), name="hall-ids",),
+    path("status/", cache_page(30)(Status), name="status",),
```

Comment:
I think this route is hit everytime someone opens up laundry on the app. If so, I think it makes sense for it to be briefly cached, but I'll ask Jordan about it.

Reply:
To my knowledge I remember that iOS caches these kind of things on their side. And if they don't, I think design-wise it would be best of this kind of caching is done on frontend side just because this route should be precise and a direct representation of what the laundry system is